.github/ISSUE_TEMPLATE/simeng-instruction-NYI-or-Alias.md

```diff
@@ -0,0 +1,51 @@
+---
+name: SimEng Instruction Not-Yet-Imlpemented, Alias, or Missing System Register
+about: Report an instruction which either does not yet have execution logic implemented (NYI), an instruction who's alias needs resolving, or an instruction which targets a system register which is not yet supported.
+title: '[NYI|ALIAS|SYSREG]'
+labels: ''
+assignees: ''
+
+---
+
+**Check List**
+ [ ] I have selected the appropriate title prefix from the available options within `[NYI|ALIAS|SYSREG]`, and added a suitable title after this.
+ [ ] The binary I am trying to run has been compiled statically for either RV64 or AArch64.
+ [ ] The compiled binary is a Linux Elf file.
+
+**Binary Compilation Instructions**
+Provide a bullet list of how the binary in question was compiled, including all compiler flags used.
+
+**SimEng CMAKE Options Used**
+Provide a bullet list of all CMAKE options used. E.g. `-DCMAKE_BUILD_TYPE=Release`.
+
+**SimEng Command Line Expression**
+Provide the comman line expression used to run SimEng e.g. `./simeng /path/to/configs/a64fx.yaml /path/to/myBinary.elf`
+
+**Console Printout**
+Please provide a console printout of all SimEng output prior to the execution statistics. Please do not include any binary specific output.
```

Comment:
Doesn't read right to me. Perhaps "Please provide a copy of SimEng's console printout prior to its execution statistics"?